cmd/logs/main.go

```diff
@@ -3,24 +3,29 @@ package main
 import (
 	"context"
 	"flag"
+	"fmt"
 	"log"
 	"os"
 
-	"github.com/pivotal/kpack/pkg/logs"
-
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/client-go/tools/clientcmd/api"
+
+	"github.com/pivotal/kpack/pkg/client/clientset/versioned"
+	"github.com/pivotal/kpack/pkg/logs"
 )
 
 var (
-	kubeconfig = flag.String("kubeconfig", "", "Path to a kubeconfig.")
+	kubeconfig = flag.String("kubeconfig", "/Users/matthewmcnew/.kube/config", "Path to a kubeconfig.")
```

Comment:
Just don't want to forget to clean this up before we merge 😄

Reply:
Ahh! Good catch!